python/conf_validation.py

```diff
@@ -0,0 +1,89 @@
+# Entry point to validating NGen catchment,nexus, and realization files
+import sys, json
+from ngen.config.realization import NgenRealization
+from ngen.config.hydrofabric import NGenCatchmentFile, NGenNexusFile
+
+def main(catchment_file,catchment_subset,nexus_file,nexus_subset,rel_file):
```

Comment:
I think in the future we will want to turn this into a little CLI tool that has documentation and what not, but for now I don't think it is worth the effort.